components/src/core/components/Input/Autocomplete/AutocompleteInput.vue

```diff
@@ -3,7 +3,7 @@
     <oxd-autocomplete-text-input
       v-bind="$attrs"
       :clear="showClear"
-      :placeholder="placeholder"
+      :placeholder="$t(placeholder)"
```

Comment:
`translateMixin` is not imported? how is this working?

Reply:
imported translateMixin